airflow/contrib/operators/bigquery_check_operator.py

```diff
@@ -171,13 +172,15 @@ class BigQueryIntervalCheckOperator(IntervalCheckOperator):
 
     @apply_defaults
     def __init__(self,
-                 table,
-                 metrics_thresholds,
-                 date_filter_column='ds',
-                 days_back=-7,
-                 gcp_conn_id='google_cloud_default',
-                 bigquery_conn_id=None,
-                 use_legacy_sql=True, *args, **kwargs):
+                 table: str,
+                 metrics_thresholds: dict,
+                 date_filter_column: str = 'ds',
+                 days_back: SupportsAbs = -7,
```

Comment:
Can't it be `int`? Personally I think it would be best to pass positive number because "days_back=-7" sounds like something in future (sorry, I am mathematician 🤷🏼‍♂️)

Reply:
It could be `int` however in this particular case I think it should be `SupportsAbs[int]` as it is in `IntervalCheckOperator` parent class. I agree about ambiguous `-7` however type `SupportsAbs` suggests that absolute value of given `int` will be taken into account.